spec/02-integration/02-cmd/11-config_spec.lua

```diff
@@ -62,16 +68,25 @@ describe("kong config", function()
 
     finally(function()
       os.remove(filename)
+      os.remove(dns_hostsfile)
     end)
 
-    helpers.start_kong({
+    assert(helpers.start_kong({
       nginx_conf = "spec/fixtures/custom_nginx.template",
-    })
+      dns_hostsfile = dns_hostsfile,
+      anonymous_reports = "on",
+    }))
+
+    local thread = helpers.udp_server(constants.REPORTS.STATS_PORT)
 
     assert(helpers.kong_exec("config db_import " .. filename, {
       prefix = helpers.test_conf.prefix,
     }))
 
+    local _, res = assert(thread:join())
+    assert.matches("signal=dbless%-import", res)
```

Comment:
FYI: here, you could use a more readable expected string like so:
```lua
assert.matches("signal=dbless-import", res, nil, true)
```

It respects the `string.match` arguments.